app/api/utils.py

```diff
@@ -385,6 +385,26 @@ def parse(self, file):
         if data:
             yield data
 
+class SpacyParser(FileParser):
+
+    def parse(self, file):
+        file = io.TextIOWrapper(file, encoding='utf-8')
```

Comment:
In https://github.com/doccano/doccano/pull/399 we introduced `EncodedIO`. Please also add this here before calling `TextIOWrapper`.